gallery/src/styles/NxBtn/NxBtnSmallExample.html

```diff
@@ -0,0 +1,25 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<div id="nx-button-small-example">
+  <div class="nx-btn-bar">
+    <button class="nx-btn nx-btn--primary nx-btn--small">Primary button</button>
+    <button class="nx-btn nx-btn--secondary nx-btn--small">Secondary button</button>
+    <button class="nx-btn nx-btn--tertiary nx-btn--small">Tertiary button</button>
+    <button class="nx-btn nx-btn--error nx-btn--small">Error button</button>
+    <button class="nx-btn nx-btn--small" disabled>Small button disabled</button>
+  </div>
+  <div class="nx-btn-bar">
+    <button class="nx-btn nx-btn--tertiary nx-btn--small" id="nx-button-small-icon-button">
+      <span>Button with icon</span><!--
+ --><svg class="nx-icon" style="height: 1em; width: 1em;" viewBox="-1 -1 2 2">
```

Comment:
This looks like the correct indentation
```suggestion
   --><svg class="nx-icon" style="height: 1em; width: 1em;" viewBox="-1 -1 2 2">
```